public/stylesheets/app/editor.less

```diff
@@ -504,3 +504,8 @@
 	height: auto;
 	border-bottom: 1px solid @modal-header-border-color;
 }
+
+// Widen autocomplete popup
+.ace_autocomplete {
+	width: 380px !important;
```

Comment:
I had created another ticket yesterday to look into this, but having had a chance to ponder the issue it might be reasonably straight-forward, will try it today.